spec/requests/user/user_profile_spec.rb

```diff
@@ -93,6 +93,19 @@
       expect(response.body).to include "M18.364 17.364L12 23.728l-6.364-6.364a9 9 0 1112.728 0zM12 13a2 2 0 100-4 2 2 0"
     end
 
+    it "displays social image if present" do
+      user.profile.update(social_image: "https://socialimage.com/image.png")
+      get "/#{user.username}"
+      # Does not include the word, but does include the SVG
```

Comment:
Misleading comment: This isn't an SVG. I'm also not sure what word isn't being included...

Reply:
@klardotsh sorry I think these are copy/pasted comments 🙃